### ISSUE TITLE
WOW as an AMD module / requirejs

### ISSUE THREAD

**Issue by khalwat** [matthieua/WOW#204]
I have a site that uses RequireJS for all javascript module loading; I've attempted to use WOW on the site, but it appears that the script isn't AMD-compliant?

Even with a shim created, what happens is the site loads, wow.js appears to load properly, I initialize it with:

    new WOW().init();

...but animations just never happen.

**Comment by attilaolah** [comment]
Possible duplicate of #155?

**Comment by khalwat** [comment]
Yes, it appears to be a duplicate.  Do you have any plans to add AMD support?

**Comment by khalwat** [comment]
http://ifandelse.com/its-not-hard-making-your-library-support-amd-and-commonjs/

**Comment by attilaolah** [comment]
Could you please try the [amd branch][1] and see if that works for you? If yes, we could merge that.

[1]: https://github.com/matthieua/WOW/tree/amd

**Comment by ZanetaBulawova** [comment]
There is an error on this line. And i tried this on common js.
https://github.com/matthieua/WOW/blob/amd/src/wow.coffee#L114
